etp-backend/src/main/clj/solita/etp/service/energiatodistus.clj

```diff
@@ -300,29 +300,24 @@
   (match/match
     [(tila-key tila-id) rooli laskutettu?]
     [:draft :laatija false] (dissoc energiatodistus-update :kommentti)
-    [:signed :laatija false]
-    (select-keys energiatodistus-update
-                 [:laskutettava-yritys-id
-                  :laskuriviviite
-                  :pt$rakennustunnus])
-    [:signed :laatija true]
-    (select-keys energiatodistus-update
-                 [:pt$rakennustunnus])
-    [:signed :paakayttaja false]
-    (select-keys energiatodistus-update
-                 [:laskutettava-yritys-id
-                  :laskuriviviite
-                  :pt$rakennustunnus
-                  :korvattu-energiatodistus-id
-                  :kommentti])
-    [:signed :paakayttaja true]
-    (select-keys energiatodistus-update
-                 [:pt$rakennustunnus
-                  :korvattu-energiatodistus-id
-                  :kommentti])
-    [:replaced :paakayttaja _]
-    (select-keys energiatodistus-update
-                 [:kommentti])
+    [:signed :laatija false] (select-keys energiatodistus-update
```

Comment:
no changes here?

Reply:
Weird. I wonder why github highlights this.